assets/src/pages/room/RoomPage.tsx

```diff
@@ -98,17 +100,34 @@ const RoomPage: FC<Props> = ({
     messageComparator
   );
 
+  const [isSidebarOpen, setIsSidebarOpen] = useState(false);
+
   return (
     <PageLayout>
       <div className="flex h-full w-full flex-col gap-y-4">
         {/* main grid - videos + future chat */}
-        <section className="flex h-full w-full flex-col">
+        <section
+          className={clsx(
+            "flex h-full w-full self-center justify-self-center 3xl:max-w-[1728px]",
+            isSidebarOpen && "gap-x-4"
+          )}
+        >
           <VideochatSection
             peers={peerState.remote}
             localPeer={peerState.local}
             showSimulcast={showSimulcastMenu}
             webrtc={webrtc}
           />
+
+          <div
+            className={clsx(
+              isSidebarOpen ? "max-w-[300px]" : "max-w-0",
+              "overflow-hidden transition-all duration-300",
```

Comment:
https://user-images.githubusercontent.com/58188916/220871689-ebb48786-a6ca-4f1e-a71d-152fdb80e0ec.mov

The transition looks fine by itself but it makes [*a dramatic zoom in*](https://youtu.be/SOxutBMCOUc?t=8) on almost every video grid layout. I think it is a little distracting but it should be discussed with others whether we want to leave this animation, make a workaround for this effect or leave it as it is.

Reply:
I think it's not related to animation but to the change of width that is occupied by the grid.

Reply:
I still think it is strange to look at. Google meet has a very nice animation for that and a very thoughtful design for this. I think we should consider developing something similar in future.